### PR TITLE
doc: terminology cleanup in usb doc

### PR DESCRIPTION
**Pull request by amyreye**
- Replace SOS or Service OS with Service VM
- Replace UOS or User OS with User VM
- Clean up some of the grammar

Signed-off-by: Amy Reyes <amy.reyes@intel.com>

### CODE REVIEW

File: doc/developer-guides/hld/usb-virt-hld.rst [hunk 1]
```diff
@@ -21,25 +21,27 @@ emulation of three components, described here and shown in
...
+  ``/sys/class/usb_role/intel_xhci_usb_sw/role``. If the user echos the string
+  ``device`` to the role node, the USB PHY will be connected with the xDCI
+  controller as
+  device mode. Similarly, by echoing ``host``, the USB PHY will be
+  connected with the xHCI controller as host mode.
+
+An xHCI register access from a User VM will induce an EPT trap from the User VM
+to
 DM, and the xHCI DM or DRD DM will emulate hardware behaviors to make
 the subsystem run.
```
Comment (dbkinder) [inline doc/developer-guides/hld/usb-virt-hld.rst:47]: @NanlinXie The table below says it's current as of ACRN 1.4.  Can you get us an update that's good for v2.7?

**Review by dbkinder — APPROVED** [review]
LGTM, but that reference to "current as of ACRN 1.4" could use updating

File: doc/developer-guides/hld/usb-virt-hld.rst (hunk 1 above)
Comment (dbkinder) [reply doc/developer-guides/hld/usb-virt-hld.rst:47]: @NanlinXie who in engineering can help here?

Comment (amyreye) [reply doc/developer-guides/hld/usb-virt-hld.rst:47]: @NanlinXie said: [Nanlin] I suggest to remove this section since it should not exit in HLD.

**Pull request merged**